pkg/metrics/collector/snapshotter.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022. Nydus Developers. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package collector
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/containerd/containerd/log"
+	"github.com/containerd/nydus-snapshotter/pkg/metrics/data"
+)
+
+type SnapshotterMetricsCollector struct{}
+type SnapshotterCacheUsageCollector struct{}
+
+var GlobalSnapshotterMetricsCollector *SnapshotterMetricsCollector
+var GlobalSnapshotterCacheUsageCollector *SnapshotterCacheUsageCollector
+
+func init() {
+	GlobalSnapshotterMetricsCollector = &SnapshotterMetricsCollector{}
+	GlobalSnapshotterCacheUsageCollector = &SnapshotterCacheUsageCollector{}
+}
+
+type SnapshotterMethod string
+
+const (
+	SnapshotterMethodUnknown SnapshotterMethod = "UNKNOWN"
+	SnapshotterMethodPrepare SnapshotterMethod = "PREPARE"
+	SnapshotterMethodMount   SnapshotterMethod = "MOUNT"
+	SnapshotterMethodRemove  SnapshotterMethod = "REMOVE"
+)
+
+func (s *SnapshotterMetricsCollector) Collect(snapshotID string, begin, end time.Time, method SnapshotterMethod) {
```

Comment:
No need to paas end time. Just get the current time as the end time in this function. It will be neater

Reply:
Fixed.